README.md

```diff
@@ -893,7 +896,7 @@ In addition to the [standard mark options](#marks), the following optional chann
 
 If **x2** is not specified, it defaults to **x1**. If **y2** is not specified, it defaults to **y1**. These defaults facilitate sharing *x* or *y* coordinates between the baseline and topline. See also the implicit stack transform and shorthand **x** and **y** options supported by [Plot.areaY](#plotareaydata-options) and [Plot.areaX](#plotareaxdata-options).
 
-By default, the data is assumed to represent a single series (a single value that varies over time, *e.g.*). If the **z** channel is specified, data is grouped by *z* to form separate series. Typically *z* is a categorical value such as a series name. If **z** is not specified, it defaults to **fill** if a channel, or **stroke** if a channel.
+By default, the data is assumed to represent a single series (a single value that varies over time, *e.g.* the closing value at the stock exchange). If the **z** channel is specified, data is grouped by *z* to form separate series. Typically *z* is a categorical value such as a series name. If **z** is not specified, it defaults to **fill** if a channel, or **stroke** if a constant.
```

Comment:
The first change is correct, but the second is not. *z* is only activated if fill or stroke is a channel.

Reply:
I think that should have been “(i.e., a single value that varies over time)” instead of e.g. And if we want to include an additional example with our rephrasing, the example should emphasize the single-series-ness, so something like “the closing price of a particular stock at the stock exchange”.

Reply:
I have to be frank, I was not at all sure about the second so I guessed to trigger some rephrasing on your side ;-)